website/src/pages/en/subgraphs/querying/managing-api-keys.mdx

```diff
@@ -1,34 +1,86 @@
 ---
-title: Managing API keys
+title: How to Manage API keys
 ---
 
+This guide shows you how to create, manage, and secure API keys for your [Subgraphs](/subgraphs/developing/subgraphs/).
+
 ## Overview
 
-API keys are needed to query Subgraphs. They ensure that the connections between application services are valid and authorized, including authenticating the end user and the device using the application.
+API keys are required to query Subgraphs. They authenticate users and devices, authorize access to specific endpoints, enforce rate limits, and enable usage tracking across The Graph.
+
+## Prerequisites
+
+- A [Subgraph Studio](https://thegraph.com/studio/) account
+
+## Create a New API Key
+
+1. Navigate to [Subgraph Studio](https://thegraph.com/studio/)
+2. Click the **API Keys** tab in the navigation menu
+3. Click the **Create API Key** button
+
+A new window will popup:
+
+4. Enter a name for your API key
+5. Optional: You can enable a period spending limit
+6. Click **Create API Key**
+
+![Create API Key Window](/img/create-api-key.png)
+
+## Manage API Keys
+
+The “API keys” table lists existing API keys and allows you to manage or delete them. For each key, you can see its status, the cost for the current period, the spending limit for the current period, and the total number of queries.
+
+### How to Set Spending Limits
+
+1. Find your API key in the API keys table
+2. Click the "three dots" menu next to the key
```

Comment:
```suggestion
2. Click the "three dots" icon next to the key
```

Reply:
Alternatively, "button". But menu is the thing that opens when clicking on the button.